core/templates/dev/head/App.ts

```diff
@@ -24,7 +24,8 @@ var oppia = angular.module(
     'angularAudioRecorder', 'dndLists', 'headroom', 'infinite-scroll',
     'ngAnimate', 'ngAudio', 'ngCookies', 'ngImgCrop', 'ngJoyRide', 'ngMaterial',
     'ngResource', 'ngSanitize', 'ngTouch', 'pascalprecht.translate', 'toastr',
-    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate'
+    'ui.bootstrap', 'ui.sortable', 'ui.tree', 'ui.validate',
+    'collectionEditorModule'
```

Comment:
Make the module name as `collectionEditorPageModule`.

Reply:
Done